client/extensions/wp-super-cache/state/stats/schema.js

```diff
@@ -4,7 +4,29 @@ export const statsSchema = {
 	patternProperties: {
 		// Site Id
 		'^\\d+$': {
-			type: 'object'
+			type: 'object',
+			items: {
+				supercache: {
+					type: 'object',
+					items: {
+						cached: { type: 'integer' },
+						cached_list: { type: 'object' },
+						expired: { type: 'integer' },
+						expired_list: { type: 'object' },
+						fsize: { type: 'integer' },
+					}
+				},
+				wpcache: {
+					type: 'object',
+					items: {
+						cached: { type: 'integer' },
+						cached_list: { type: 'object' },
+						expired: { type: 'integer' },
+						expired_list: { type: 'object' },
+						fsize: { type: 'integer' },
+					}
```

Comment:
Since `supercache` and `wpcache` have the exact same shape, you could consider defining a `cacheScheme` const to re-use here.